tools/rag/query.py

```
@@ -0,0 +1,196 @@
+            info = client.get_collection(c.name)
+            print(f"  - {c.name}: {info.points_count} vectors")
+        return
+
+    # Show stats
+    if args.stats:
+        try:
+            info = client.get_collection(args.collection)
+            print(f"Collection: {args.collection}")
+            print(f"  Vectors: {info.points_count}")
+            print(f"  Status: {info.status}")
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)